octavia-cli/octavia_cli/apply/resources.py

```diff
@@ -573,6 +586,46 @@ def _create_configured_catalog(sync_catalog: dict) -> AirbyteCatalog:
             )
         return AirbyteCatalog(streams_and_configurations)
 
+    def _deserialize_operations(
+        self, operations: List[dict], outputModelCls: Union[Type[OperationCreate], Type[WebBackendOperationCreateOrUpdate]]
```

Comment:
Is `outputModelCls` supposed to mean "output model class"? If so could we rename this parameter to `outputModelClass`? I was confused by `Cls`

Reply:
Done